packages/babel-plugin-proposal-class-properties/test/fixtures/nested-class/super-call-in-decorator/output.js

```diff
@@ -16,7 +16,7 @@ let Outer = /*#__PURE__*/function (_Hello) {
     var _this;
 
     babelHelpers.classCallCheck(this, Outer);
-    _dec = _this = _super.call(this)
+    _dec = _this = _super.call(this);
```

Comment:
I don't know why this line is changed after rebasing. Maybe a generator update?

Reply:
https://github.com/babel/babel/pull/14398 probably